### PR TITLE
2 task

### PR DESCRIPTION
**Pull request by AlexanderChechyotko**
add changes in task 2

### CODE REVIEW

File: 2/css/style.css [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+}
+
+.b-red{
+    margin-bottom: 20px;
+    height: 80px;
+    background: #ff4242;
+}
+
+
+.margin-container{
```
Comment (EvgenyRazumov) [inline 2/css/style.css:28]: Я бы не именовал класс таким образом: именование должно происходить по смыслу, а не по вложенному стилю.
Пример: предположим, что для этого элемента тебе нужно будет поменять стиль с margin на padding. При этом тебе придется менять имя класса и html код. Если бы ты дал более нейтральное название css-классу, этого делать бы не нужно было.

File: 2/index.html [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+            <div class="col-md-4"><div class="margin-container"><div class="b-green"></div></div></div>
+        </div>
+        <div class="row"> 
+            <div class="col-md-6 col-lg-3 "><div class="margin-container"><div class="b-red"></div></div></div>
+            <div class="col-md-6 col-lg-3 "><div class="margin-container"><div class="b-red"></div></div></div>
+            <div class="col-md-6 col-lg-3 "><div class="margin-container"><div class="b-red"></div></div></div>
+            <div class="col-md-6 col-lg-3 "><div class="margin-container"><div class="b-red"></div></div></div>
+        </div>
+    </div>
+    <script src="https://ajax.googleapis.com/ajax/libs/jquery/1.11.3/jquery.min.js"></script>
```
Comment (EvgenyRazumov) [inline 2/index.html:55]: Файлы со скриптами плагинов следует подключать в секции head. Перенеси это пожалуйста.

**Comment by EvgenyRazumov** [comment]
И еще один общий момент, касающийся данного задания: структуру папок со стилями и со скриптами стоило бы поменять.
Что я имею ввиду: все css стили я бы хранил в папке css, все скрипты в папке js. Это бы облегчило мне поиск по файлам в будущем. У тебя же получается что в папке css лежит только твой стиль, а стили бутстрапа уже в отдельной папке.
